spec/models/user_spec.rb

```diff
@@ -144,14 +145,14 @@
     end
 
     it "is false when the ect profile is withdrawn" do
-      user = create(:early_career_teacher_profile, status: "withdrawn").user
+      user = create(:participant_profile, :ect, status: "withdrawn").user
```

Comment:
Consistency please, either symbols everywhere or not at all

Reply:
Would prefer a trait, tbh :)